tasklist/webapp/src/test/java/io/camunda/tasklist/webapp/es/backup/es/BackupManagerElasticsearchTest.java

```diff
@@ -45,19 +53,21 @@ class BackupManagerElasticsearchTest {
 
   @InjectMocks private BackupManagerElasticSearch backupManager;
 
-  @Mock
+  @Mock(strictness = Strictness.LENIENT)
```

Comment:
❓ What does this?

Reply:
it does not fail the test if a mocked method is not invoked. I add to add it because I moved up all common mocks in a `@BeforeEach` method

Reply:
Maybe i get it wrong but wouldnt this make verifications and order checks useless? 🤔

Reply:
The default strictness is to avoid having too many stubs that are not needed. 
You don't usually verify all the stubs you make (for example if you are stubbing a response from a service), but only some of them :man_shrugging: